### PR TITLE
Text copy

### PR DESCRIPTION
**Pull request by shanitajohn**

### CODE REVIEW

File: app/views/pages/review_fixture.html [hunk 1]
```diff
@@ -1781,7 +1781,7 @@
 </p>
   <div id="submission-agreement" class="">
       <p id="submission-agreement-copy">
-        I hereby grant to Emory University and its agents....
+        I hereby grant to Emory University and its agents the non-exclusive license to archive, make accessible, and display, subject to any embargo restrictions I have specified above, my thesis or dissertation in whole or in part in all forms of media, now or hereafter known, including the display of the thesis or dissertation on the world wide web. I retain all ownership rights to the copyright of the thesis or dissertation. I also retain the right to use in future works (such as articles or books) all or part of this thesis or dissertation. I certify that my electronic submission is the version of my thesis/dissertation that was approved by my committee.
```
Comment (jenlindner) [inline app/views/pages/review_fixture.html]: This files shouldn't be here, I'm sorry. I'll remove it.

**Comment by jenlindner** [comment]
@shanitajohn, do you know about checking out the details of failing tests in Travis? The link will take you to a page showing the server log of running the tests. All you have to do, if you see a detail that you want to change, is go into your code on your local machine again, make your changes, add and commit them, rebase again, and when you push again add a -f. That -f means 'force push' and it will kick off the Travis tests again. If you want to talk about it at all ping me :-)

**Comment by bess** [comment]
I'm working through this and re-submitting it in bits.

**Comment by bess** [comment]
Resubmitted elsewhere.

**Pull request closed without merging**